azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -154,6 +159,28 @@ public ExecutorManager(final Props azkProps, final ExecutorLoader loader,
     this.cleanerThread = new CleanerThread(executionLogsRetentionMs);
     this.cleanerThread.start();
 
+    this.globalProps = setupGlobalProps(azkProps);
+  }
+
+  private Props setupGlobalProps(Props azkProps) {
+    String globalPropsPath = azkProps.getString("executor.global.properties", null);
+    if (Objects.nonNull(globalPropsPath)) {
```

Comment:
`Objects.nonNull` isn't a typical way that we check if a variable is null.

Maybe we can just use `globalPropsPath != null`?